website/client/src/assets/scss/dropdown.scss

```diff
@@ -54,8 +55,8 @@
 
 
   &:active, &:hover, &:focus, &.active {
```

Comment:
@negue I made some changes to this in develop, the rules here are lines 58-59 and 66-67 should get an !important or the user dropdown won't display correctly

Reply:
applied the `!importants` from develop